### ISSUE TITLE
Organization Update doesn't work without updating Display Image

### ISSUE THREAD

**Issue by prathamesh-mutkure** [PalisadoesFoundation/talawa-admin#818]
**Describe the bug**

- The organization update fails
- Because the Image is missing

**To Reproduce**

1. Go to an organization dashboard
2. Go to setting through Navbar
3. Click on update organization 
4. Update any one - name, location, or description
5. Click save changes

**Expected behavior**

- The updated organization details should get saved

**Actual behavior**

- It returned a 422 error stating - `Invalid file type`

**Screenshots**

Error when changing just the location -

<img width="1904" alt="Screenshot 2023-03-28 at 5 06 36 PM" src="https://user-images.githubusercontent.com/28570857/228224034-4637a881-efef-4c16-a013-a7e37b447b1a.png">

**Additional details**

- This issue gets fixed if we also select a new display image 

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by prathamesh-mutkure** [comment]
Hi @palisadoes, I'd like to resolve this

**Comment by palisadoes** [comment]
This issue isn't fixed. Tests were failing on the push.

- https://github.com/PalisadoesFoundation/talawa-admin/actions/runs/4622064669/attempts/1

**Comment by prathamesh-mutkure** [comment]
@palisadoes I'll fix it

**Comment by prathamesh-mutkure** [comment]
@palisadoes the tests seemed to have passed on second attempt?

- https://github.com/PalisadoesFoundation/talawa-admin/actions/runs/4622064669

**Comment by prathamesh-mutkure** [comment]
I guess we'll have to revert the "revert commit"

**Comment by palisadoes** [comment]
No. It means that there is an intermittent bug that needs to be rectified.

**Comment by prathamesh-mutkure** [comment]
I actually tried running all the test cases on my local and it seems to pass, it there something I'm missing? 

<img width="997" alt="Screenshot 2023-04-06 at 2 18 40 AM" src="https://user-images.githubusercontent.com/28570857/230208013-fa22efb4-f295-4463-90ff-5e8f10e69993.png">

I've experienced this issue before too when I pushed an additional commit while the workflow for the last PR was still running, can that be the case here? #840 was merged immediately after, maybe that's causing the issue?

Can we try merging once more through a new PR?
